WooCommerce/Classes/ViewModels/Order Details/Aggregate Order Items/AggregateOrderItem.swift

```diff
@@ -55,7 +55,6 @@ extension AggregateOrderItem: Hashable {
     public func hash(into hasher: inout Hasher) {
         hasher.combine(productID)
         hasher.combine(variationID)
-        hasher.combine(attributes)
```

Comment:
I went looking through the code for why this was added... unfortunately, removing it causes regressions.

- #4445

More details in the main review comment.